src/geometry/manifold/algebra/smooth_functions.lean

```diff
@@ -80,6 +89,17 @@ instance smooth_map_group {G : Type*} [group G] [topological_space G]
   mul_left_inv := λ a, by ext; exact mul_left_inv _,
```

Comment:
Something like
```suggestion
  mul_left_inv := λ a, by ext; exact mul_left_inv _,
  div := λ f g, ⟨λ x, (f x / g x), f.smooth.div g.smooth⟩
```

Reply:
I only just saw this. I think this is the right approach but will require creation of `smooth.div`.

Reply:
I have now attempted something along these lines with #6912